api/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
+
 # Create your models here.
+
+
+class Bracket(models.Model):
+    bracket_id = models.CharField(max_length=100)
```

Comment:
We should get the bracket id by default. I don't believe that we need to define it here.